.circleci/test-deploy.yml

```diff
@@ -0,0 +1,147 @@
+---
+version: 2.1
+
+# ----------------
+# Orbs declaration
+# ----------------
+
+orbs:
+  noos-ci: noosenergy/noos-ci@dev:<<pipeline.git.revision>>
+  orb-tools: circleci/orb-tools@12.0.4
+
+release-filters: &release-filters
+  tags:
+    # ONLY git tag compliant format: e.g. `major-release-1.0.0`
+    only: /^(major|minor|patch)-release-v\[0-9]+(\.[0-9]+)*(-alpha\.[0-9]+)?/
```

Comment:
As discussed, better to align our versioning syntax across our repositories (such a syntax was a legacy when the orb tools were first released).

https://github.com/noosenergy/noos-invoke/blob/master/.circleci/config.yml#L111

```
```